src/sentry/tsdb/snuba.py

```diff
@@ -21,19 +32,72 @@ class SnubaTSDB(BaseTSDB):
 
     # The ``model_columns`` are translations of TSDB models into the required
     # columns for querying snuba. Keys are ``TSDBModel`` enumeration values,
-    # values are in the form ``(groupby_column, aggregateby_column or None)``.
+    # values are ``SnubaModelSettings``s.
```

Comment:
This comment is not very clear now that the implementation has changed, and I think `_columns` is not a very representative suffix at this point, since `SnubaModelSettings` contains more than just column references (dataset and conditions.)